frontend/src/components/WaitingRoom/BlurButton/BlurButton.tsx

```diff
@@ -35,7 +35,6 @@ const BlurButton = (): ReactElement | false => {
         <Tooltip title={title} aria-label="toggle background blur">
           <VideoContainerButton
             onClick={toggleBlur}
-            isEnabled={!hasBlur}
```

Comment:
Was this intentional?  Seeing other examples of similar removals as well.

Reply:
yes! please see [this comment](https://github.com/Vonage/vonage-video-react-app/pull/143/files#r2045236051) 🙏